pyproject.toml

```diff
@@ -22,6 +22,7 @@ redis = "^4.1.0"
 django-celery-beat = "^2.4.0"
 python-dateutil = "^2.8.2"
 django-cors-headers = "^4.3.1"
+django-guardian = "^2.4.0"
```

Comment:
oh cool, didn't know something like this existed

Reply:
I remember researching it when @rgossiaux brought it up in https://github.com/cardinalitypuzzles/cardboard/issues/191

Should we be concerned that the codebase hasn't been touched in 2+ years? https://github.com/django-guardian/django-guardian

Reply:
On that note, is there a reason to use `django-guardian` over the per-object permissions provided by DRF? https://www.django-rest-framework.org/api-guide/permissions/#object-level-permissions

Reply:
as far as I know it's the most popular/mature library for this in the django ecosystem. No commits in 2 years is probably a bit worrying but I'm just doing pretty basic core stuff with it so I'm personally not too bothered atm

DRF provides view methods for checking permissions (I'm using them in this PR already I think), but not a permissions backend itself. Still need some way of storing permissions in the DB--that's what django-guardian provides